Transportation/ServiceNetwork/Route/doc/spec.md

```diff
@@ -0,0 +1,48 @@
+# Route
+
+## Description
+
+This entity model a particular transport route model, including all properties which are commom to multiple itinerary instances belonging to such model. A route is a way to identify a journey that is regularly made by a given transport in an agency.
+
+## Data Model
+
+- ```id```: Entity's unique identifier.
+
+- ```type```: Entity type. It must be equal to ```Route```.
+
+- ```shortName```: Short name of a route, often a short and abstract identifier like "7", "34", or "Blue" that riders use to identify a route.
+    - Normative References: https://schema.org/name
+    - Mandatory 
+
+- ```longName```: Full name of a route, more descriptive than the ```route_short_name``` and often include the route's destination or stop
+    - Normative References: https://schema.org/name
+    - Mandatory 
+
+- ```description```: Description about this route.
+	- Attribute type: https://schema.org/Text
+	- Optional 
+
+- ```refAgency```: Agency this route operated by.
+        - Attribute type: Reference to a [Agency](../../Agency/doc/spec.md)
+        - Mandatory
+
+- ```color```: Color that corresponds to the route. The color must be provided as a six-character hexadecimal number, for example, 00FFFF.
+	- Attribute type: [Text](https://schema.org/Text)
+	- Optional
```

Comment:
is this REALLY an attribute of a route?

Reply:
It is quire common when mapping Public Transport GTFS files which usually contain a color for each route. But it is something not really related to a route.